lucene/spatial-extras/src/java/org/apache/lucene/spatial/spatial4j/Geo3dShapeFactory.java

```diff
@@ -125,7 +125,12 @@ public Point pointXY(double x, double y) {
 
   @Override
   public Point pointXYZ(double x, double y, double z) {
```

Comment:
x,y,z here is horizontal (longitude), vertical (latitude), and some user-defined value.  It's not geo-centric coordinates as is found commonly in Geo3d.  This method is called by the WKT parser (and perhaps GeoJSON?) if the input format has a 3rd dimension to it.